splash/src/main/kotlin/edu/artic/splash/SplashActivity.kt

```diff
@@ -225,8 +226,18 @@ class SplashActivity : BaseViewModelActivity<SplashViewModel>(), TextureView.Sur
         val options = ActivityOptions
                 .makeSceneTransitionAnimation(this, textureView, "museumImage")
         if (!isFinishing) {
-            startActivity(intent, options.toBundle())
-            finishAfterTransition()
+            /**
+             * Shared transition element does not work properly (crashes on some devices) running
+             * Lollipop. Please check https://issuetracker.google.com/issues/35826109
+             */
+            if (Build.VERSION.SDK_INT < VERSION_CODES.M &&
+                    Build.VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP) {
```

Comment:
I think this is all acceptable. Although...why not just check against Marshmallow?

Reply:
Agreed. Removed the `Build.VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP` condition since min support is `VERSION_CODES.LOLLIPOP`